fabfile/tasks.py

```diff
@@ -0,0 +1,147 @@
+"""
+Fabric tasks for PyBitmessage devops operations.
+"""
+
+import os
+import sys
+
+from fabric.api import run, task, hide, cd
+from fabvenv import virtualenv
+
+from fabfile.lib import pycodestyle, flake8, pylint, autopep8, PROJECT_ROOT, VENV_ROOT, coerce_bool, flatten
+
+
+def get_tool_results(file_list):
+    """Take a list of files and resuln the results of applying the tools"""
+    results = []
+    for path_to_file in file_list:
```

Comment:
Split the function into sub functions...

Reply:
OK, fix incoming